### ISSUE TITLE
[Feature Request]: Krypton Calendar

### ISSUE THREAD

**Issue by MattH-Work** [Krypton-Suite/Extended-Toolkit#365]
How would I go about NOT showing the Start & End time in an event (the text in Red)
I would like to hide the time, therefore allowing more of the event description to be shown
![image](https://user-images.githubusercontent.com/84494183/202180942-606e74dc-7048-425b-8057-acab1dd956d4.png)

**Comment by Wagnerp** [comment]
Hi @MattH-Work, 

Before I start investigating, can you confirm that you're using the `Krypton.Toolkit.Suite.Extended.Calendar` module?

**Comment by MattH-Work** [comment]
@Wagnerp, 

Yes I can confirm, I am using the Krypton.Toolkit.Suite.Extended.Calendar module
